yarn.lock

```diff
@@ -3315,6 +3315,14 @@
     "@typescript-eslint/types" "5.49.0"
     "@typescript-eslint/visitor-keys" "5.49.0"
 
+"@typescript-eslint/scope-manager@5.50.0":
```

Comment:
no changes to package.json so this `yarn.lock` change seems stray

Reply:
I did add `@agoric/vat-data` to `packages/vats/package.json` in another commit.

I do lack a good mental model of what changed to expect in `yarn.lock`.

Reply:
oh yeah. still shouldn't change the lockfile. best to take it out of the PR. I confirmed it doesn't need changing.
```
gco origin/6553-board-upgrade
gco master yarn.lock
yarn install
git diff
```